bitnami/elasticsearch/README.md

```diff
@@ -50,407 +50,431 @@ $ helm delete --purge my-release
-| `volumePermissions.image.tag`                     | Init container volume-permissions image tag                                                                                                                       | `"10"`                                                       |
-| `volumePermissions.image.pullPolicy`              | Init container volume-permissions image pull policy                                                                                                               | `Always`                                                     |
-| `volumePermissions.resources`                     | Init container resource requests/limit                                                                                                                            | `requests: {}, limits: {}`                                   |
-
-### Kibana Parameters
-
-| Parameter                    | Description                                                               | Default                                                                                 |
-|------------------------------|---------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
-| `global.kibanaEnabled`       | Use bundled Kibana                                                        | `false`                                                                                 |
-| `kibana.elasticsearch.hosts` | Array containing hostnames for the ES instances. Used to generate the URL | `{{ include "elasticsearch.coordinating.fullname" . }}` Coordinating service (fullname) |
-| `kibana.elasticsearch.port`  | Port to connect Kibana and ES instance. Used to generate the URL          | `9200`                                                                                  |
+| Name                             | Description                                 | Value                   |
+| -------------------------------- | ------------------------------------------- | ----------------------- |
+| `sysctlImage.enabled`            | Enable kernel settings modifier image       | `true`                  |
+| `sysctlImage.registry`           | Kernel settings modifier image registry     | `docker.io`             |
+| `sysctlImage.repository`         | Kernel settings modifier image repository   | `bitnami/bitnami-shell` |
+| `sysctlImage.tag`                | Kernel settings modifier image tag          | `10-debian-10-r127`     |
+| `sysctlImage.pullPolicy`         | Kernel settings modifier image pull policy  | `Always`                |
+| `sysctlImage.pullSecrets`        | Kernel settings modifier image pull secrets | `[]`                    |
+| `sysctlImage.resources.limits`   | The resources limits for the container      | `{}`                    |
+| `sysctlImage.resources.requests` | The requested resources for the container   | `{}`                    |
```

Comment:
This doesn't look like a `VolumePermissions` section

Reply:
Indeed, copy-paste error as always. Fixed in 58b852f. Thanks!